playwright-e2e/tests/osteosarcoma/adult-enrollment.spec.ts

```diff
@@ -347,14 +348,20 @@ test('Osteo adult self enroll @osteo', async ({ page }) => {
   });
 
   await familyHistoryPage.next();
+
+  await new TextArea(page, { ddpTestID: 'answer:FH_OTHER_FACTORS_CANCER_RISK' }).toLocator().waitFor({
+    state: 'visible',
+    timeout: 60 * 1000
+  });
```

Comment:
Looking for a way to determine whether page has completed rendering, this textarea is visible after rendering.